lib/ansible/modules/network/ovs/openvswitch_bridge.py

```diff
@@ -123,12 +125,23 @@ def map_obj_to_commands(want, have, module):
             commands.append(command)
     else:
         if have:
-            if want['fail_mode'] != have['fail_mode']:
-                templatized_command = ("%(ovs-vsctl)s -t %(timeout)s"
-                                       " set-fail-mode %(bridge)s"
-                                       " %(fail_mode)s")
-                command = templatized_command % module.params
-                commands.append(command)
+            wanted_fail_mode = want['fail_mode']
+            have_fail_mode = have['fail_mode']
+            if wanted_fail_mode != have_fail_mode:
+                if wanted_fail_mode is not None and wanted_fail_mode != 'None':
```

Comment:
Please merge this clause with previous if-clause using `and`, this will reduce the whole block indentation. Less nesting is better.